clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1030,8 +1030,9 @@ template <typename T> class trace_converter_t {
             TESTANY(OFFLINE_FILE_TYPE_FILTERED, impl()->get_file_type(tls));
```

Comment:
Also remove the ignore-list entry: https://github.com/DynamoRIO/dynamorio/blob/02b88bb247899af74f95a42b1f512c0a64070686/suite/runsuite_wrapper.pl#L348

Reply:
Done.